custom_components/wyzeapi/__init__.py

```diff
@@ -105,8 +110,8 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
             token,
         )
     except:
-        _LOGGER.error("Wyzeapi: Could not login. Please re-login through integration configuration.")
-        raise ConfigEntryAuthFailed("Unable to login, please re-login.")
+        _LOGGER.error("Wyzeapi: Could not login. Please re-login through integration configuration")
+        raise ConfigEntryAuthFailed("Unable to login, please re-login.") from None
```

Comment:
Why the `from None`?

Reply:
Its just something that the lint configuration from home assistant prefers.